test-data/apply-configuration/eventing-01/input-dir/cluster-scoped-resources/operator.openshift.io/authentications/cluster.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: operator.openshift.io/v1
+kind: Authentication
+metadata:
+  creationTimestamp: "2023-12-06T09:18:01Z"
+  generation: 1
+  name: cluster
+  resourceVersion: "1276"
+  uid: 28ab65e3-c0c5-45ac-8df3-079fd1e35f20
```

Comment:
I think we can easily loose track of how diverse these can be, unless we have a specific test for that. 
That test could have files for all supported ways of storing resources:
```
- cluster-scoped-resource/group/resource/<name>.yaml
- cluster-scoped-resource/group/resource.yaml
- namespaces/<namespace>/group/resource/<name>.yaml
- namespaces/<namespace>/group/resource.yaml
```